kamon-core/src/main/scala/kamon/Kamon.scala

```diff
@@ -155,7 +148,7 @@ object Kamon extends MetricLookup with ReporterRegistry with io.opentracing.Trac
     * was no active Span then the provided fallback value
     */
   def fromActiveSpan[T](code: ActiveSpan => T): Option[T] =
```

Comment:
Sure thing! There is nothing that we can actually read from the Span but rather the SpanContext.. before it would be nice to get the TraceID/SpanID for logging, or get some baggage key but now that will be handled differently.. cool.